feat(formatter): add option to split binary expressions before operators (#6159)
biomejs/biome: .changeset/mighty-seals-dance.md

---
"@biomejs/biome": minor
---

> Added a new option to Biome's Javascript formatter, `operator_linebreak`, to configure whether long lines should be broken before or after binary operators.

Reviewer: ```suggestion
Added a new option to Biome's JavaScript formatter, `javascript.formatter.operatorLinebreak`, to configure whether long lines should be broken before or after binary operators.
```

> For example:

Reviewer: ```suggestion
For example, the following configuration:
```

```json
{
  "formatter": {
    "javascript": {
      "operatorLinebreak": "before" // defaults to "after"
    }
  }
}
```

> Will cause this JavaScript file

Reviewer: ```suggestion
Will cause this JavaScript file:
```
